### DISCUSSION TITLE
左右のキーボードが接続されない（片方ずつしか使えない）

### DISCUSSION THREAD

**Discussion by arakiyudai** [Yowkees/keyball#673] [Q&A]
[片方のキーボードが認識されない #301](https://github.com/Yowkees/keyball/discussions/301#discussion-5511384)
こちらのQAと似た症状が出ているのですが、解決できません。サポートをお願いいたします。

### 症状
* 左右TRSケーブル接続、左にType-Cケーブル接続した状態で、右が機能しない（逆も同じ）
* Type-C繋いだ方はOLED表示、キー入力問題なし
* Type-C繋いでない方はOLED表示されない、キー入力できない、赤いインジケーター？だけついている状態

### 試したこと
* TRRSケーブルの交換　→効果なし
* TRRS端子部分のはんだづけやり直し　→効果なし
* [片方のキーボードが認識されない #301](https://github.com/Yowkees/keyball/discussions/301#discussion-5511384)で紹介されていた左右キーボードのdata端子同士の導通　→導通には問題なし（テスターで確認済み）

### 画像
![typec-left](https://github.com/user-attachments/assets/a489d25a-0c7d-4668-93a7-bf31cdfce932)
![typec-right](https://github.com/user-attachments/assets/f3605bd2-29c4-4971-b558-a1493d732e44)
![left-side](https://github.com/user-attachments/assets/0be7c1ea-2e0a-4d45-887c-d50113ade3ed)
![right-side](https://github.com/user-attachments/assets/1f27948b-b31c-4252-a6b3-6f2709ac69c2)

### 追記
![CleanShot 2024-10-13 at 00 07 02@2x](https://github.com/user-attachments/assets/683f03e4-3214-478a-b8f8-a10100f3624f)
chatGPTに聞きながら問題解決に取り組んでいます。左右キーボードのTRRS端子の導通チェックしてみろということで、左右の同じ端子同士を確認しました。4つの端子のうち3つは導通確認できましたが、添付画像オレンジの丸のVCC端子同士が導通確認できませんでした。これはそもそも導通しないものなのでしょうか？（何度ハンダ付けし直しても改善せず...）
関係あるかどうかわかりませんが、TRRSのVCC端子とPRO MicroのVCC端子同士は左右どちらも導通を確認できました。
TRRSジャックもしくはケーブルの問題なのでしょうか...？

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。回答が遅くなり申し訳ありませんでした。

追記画像のオレンジ丸で示していただいた端子はVCCになります。TRSケーブルを繋いだ状態で、オレンジ丸の２点間が導通していないようですと、ご指摘の症状となります。

写真でははっきりと分かりませんが、オレンジ丸部の半田付けを再度見直してみてください。
また、やはりTRSケーブルに問題がある気もします。再度、別のケーブルでご確認いただければ幸いです。

以上、お手数ですがご確認お願いします。
